exporter/templates/cat1/r3/_2.16.840.1.113883.10.20.24.3.47.xml

```diff
@@ -21,7 +21,7 @@
       </effectiveTime>
     {{end}}
     {{if .EntrySection.AllowedAdministrations}}
-      <repeatNumber value="{{escape .EntrySection.AllowedAdministrations}}" />
+      <repeatNumber value="{{ .EntrySection.AllowedAdministrations }}" />
```

Comment:
Can we stick to `{{no extra whitespace}}` just for consistency? Same for the other three in this PR. Sorry to be picky. This otherwise looks all good to me